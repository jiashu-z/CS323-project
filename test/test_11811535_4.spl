my_struct
{
  int code;
  char data;
};
int test_11811535_4()
{
  struct my_struct a;
}
