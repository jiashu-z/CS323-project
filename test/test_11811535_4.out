Error type B at Line 2: Missing specifier
Error type B at Line 5: Missing specifier